src/main/java/jenkins/branch/DefaultBranchPropertyStrategy.java

```diff
@@ -81,6 +82,7 @@ public DefaultBranchPropertyStrategy(@CheckForNull BranchProperty[] props) {
     /**
      * Our {@link BranchPropertyStrategyDescriptor}.
      */
+    @Symbol("allBranchesSameProperties")
```

Comment:
Any thoughts on naming here (or for `NamedExceptionsBranchPropertyStrategy`)? I think I saw that you were considering `sameProperties` elsewhere? The display names for these types in the UI are ["All branches get the same properties"](https://github.com/jenkinsci/branch-api-plugin/blob/b99766caece8edb67adc0ce2dde8e3f0d2df67c5/src/main/resources/jenkins/branch/Messages.properties#L26) and ["Named branches get different properties"](https://github.com/jenkinsci/branch-api-plugin/blob/b99766caece8edb67adc0ce2dde8e3f0d2df67c5/src/main/resources/jenkins/branch/Messages.properties#L29), so I just tried to pick names that were close to what users would be used to from the UI. Maybe `Properties` is redundant since the type is `BranchPropertyStrategy`, so we could use `allBranchesSame` and `namedBranchesDifferent` or something? IDK what is best.

Reply:
Yeah, I removed the suggestions because i thought i was overthinking it. But yeah, I thought  the wording was a bit redundant. I'm fine with `allBranchesSame` and `namedBranchesDifferent`.  I like to reduce names as much as possible, but I guess if you reduce it too much you can lose the meaning (like `same` and `differentNames`)